templates/zerver/help/add-emoji.md

```diff
@@ -5,28 +5,32 @@ shows you how to send messages containing emoji by using Zulip’s built-in emoj
 menu, pasting Unicode, or manually inputting the emoji code.
 
 ## Using the emoji menu
-1. Start by composing a [new message](/help/reply-to-a-message).
-2. Click the smiley face (<i class="icon-vector-smile"></i>) icon under the left
-edge of the message box.
-3. Click on any emoji to include it in your message.
+1. Compose a [new message or reply](/help/reply-to-a-message).
+2. Click the smiley face (<i class="icon-vector-smile"></i>) icon at the
```

Comment:
I think it might be `in the corner` in this context:
> The way I see it is that whichever way you express it, it's in the corner because it's inside the 90° angle. You only use at or on the corner when referring to the outside angle. 

source: https://forum.wordreference.com/threads/write-at-on-in-the-right-hand-corner-of-the-first-page.856697/